coininfo.js

```diff
@@ -6192,6 +6192,885 @@ const coininfo = {
+    total_supply: null,
```

Comment:
By 'null' we are saying that a coin does not have a capped max supply - that a possible max supply is infinite. An etheruem is an example of it. 
Cardano (ADA) should have max supply of 45 billion (I think). I found the same 'null' in other info of coins, please adjust total_supply field for the coins